FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/DemoTasks/FleetProvisioningDemoExample.c

```diff
@@ -0,0 +1,864 @@
+        }
+
+        /**** Disconnect from AWS IoT Core ************************************/
+
+        /* As we have completed the provisioning workflow, we disconnect from
+         * the connection using the provisioning claim credentials. We will
+         * establish a new MQTT connection with the newly provisioned
+         * credentials. */
+        if( xConnectionEstablished == true )
+        {
+            xDisconnectMqttSession();
+            xConnectionEstablished = false;
+        }
+
+        /**** Connect to AWS IoT Core with provisioned certificate ************/
+
+        if( xStatus == true )
+        {
+            LogInfo( ( "Establishing MQTT session with provisioned certificate..." ) );
+            xStatus = xEstablishMqttSession( prvProvisioningPublishCallback,
+                                           pkcs11configLABEL_DEVICE_CERTIFICATE_FOR_TLS,
+                                           pkcs11configLABEL_DEVICE_PRIVATE_KEY_FOR_TLS );
+
+            if( xStatus != true )
+            {
+                LogError( ( "Failed to establish MQTT session with provisioned "
+                            "credentials. Verify on your AWS account that the "
+                            "new certificate is active and has an attached IoT "
+                            "Policy that allows the \"iot:Connect\" action." ) );
+            }
+            else
+            {
+                LogInfo( ( "Sucessfully established connection with provisioned credentials." ) );
+                xConnectionEstablished = true;
+            }
+        }
+
+        /**** Finish **********************************************************/
+
+        if( xConnectionEstablished == true )
+        {
+            /* Close the connection. */
+            xDisconnectMqttSession();
+            xConnectionEstablished = false;
+        }
+
+        /**** Retry in case of failure ****************************************/
+
+        /* Increment the demo run count. */
+        ulDemoRunCount++;
+
+        if( xStatus == true )
+        {
+            LogInfo( ( "Demo iteration %d is successful.", ulDemoRunCount ) );
+        }
+        /* Attempt to retry a failed iteration of demo for up to #fpdemoMAX_DEMO_LOOP_COUNT times. */
+        else if( ulDemoRunCount < fpdemoMAX_DEMO_LOOP_COUNT )
+        {
+            LogWarn( ( "Demo iteration %d failed. Retrying...", ulDemoRunCount ) );
+            vTaskDelay( fpdemoDELAY_BETWEEN_DEMO_RETRY_ITERATIONS_SECONDS );
+        }
+        /* Failed all #fpdemoMAX_DEMO_LOOP_COUNT demo iterations. */
+        else
+        {
+            LogError( ( "All %d demo iterations failed.", fpdemoMAX_DEMO_LOOP_COUNT ) );
+            break;
+        }
+    } while( xStatus != true );
+
+    /* Log demo success. */
+    if( xStatus == true )
+    {
+        LogInfo( ( "Demo completed successfully." ) );
+    }
+
+    /* Delete this task. */
+    LogInfo(("Deleting Fleet Provisioning Demo task."));
```

Comment:
```suggestion
    LogInfo( ( "Deleting Fleet Provisioning Demo task." ) );
```

Reply:
Addressed in this commit: https://github.com/FreeRTOS/FreeRTOS/pull/724/commits/0a18eb6432509f18fd6b1015b07ca343ca5d4e32